src/Classes/Inflector.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Classes;
+
+use Doctrine\Inflector\InflectorFactory;
+use Doctrine\Inflector\Rules\Pattern;
+use Doctrine\Inflector\Rules\Patterns;
+use Doctrine\Inflector\Rules\Ruleset;
+use Doctrine\Inflector\Rules\Substitutions;
+use Doctrine\Inflector\Rules\Transformation;
+use Doctrine\Inflector\Rules\Transformations;
+
+/**
+ * Our own static proxy to the doctrine inflector
+ * so we only have to defines rules in a single place.
+ */
+class Inflector
```

Comment:
i don't like this idea of a static proxy at all. Proposing to make this a service that wraps around the doctrine inflector, and then inject it into all places necessary.

Reply:
I agree, but it's static now so this was the easiest way to move forward. In particular we need this in tests where we don't have a DI system. In order to work in both places the final service will almost certainly be a factory that injects an instance of the static proxy anyway.

Reply:
Took a different approach, let me know what you think.

Reply:
looks great.